12-kubernetes-cluster.tf

```diff
@@ -35,7 +35,8 @@ resource "azurerm_kubernetes_cluster" "kubernetes_cluster" {
 
   sku_tier = var.sku_tier
   default_node_pool {
-    name                 = "nodepool"
+    name                 = var.project == "ss" ? "system" : "nodepool"
```

Comment:
we shouldn't have project specific configuration in here.

we could do it by feature flagging:

```
var.enable_user_system_nodepool_split
```

or declaratively by specifying the taints and name but I don't think that's as good